web-app/stories/Button.stories.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Button } from './Button';
+
+export default {
+  title: 'Example/Button',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
  title: "Example/Button",
```